default.build

```diff
@@ -65,7 +65,7 @@
 	<target name="find-async-generator-console">
 		<property name="nuget-package-id" value="CSharpAsyncGenerator.CommandLine" />
 		<call target="get-nuget-package-path" />
-		<property name="async-generator-console"  value="${nuget-package-path}/tools/AsyncGenerator.CommandLine.exe" />
+		<property name="async-generator-console"  value="${nuget-package-path}/tools/net461/AsyncGenerator.CommandLine.exe" />
```

Comment:
`net461` is now deprecated as the new version of Roslyn supports only `net472` and `netcoreapp2.1`. Currently the generator does not support any C# 8 features yet, so there are no difference between them in terms of supported features.